### PR TITLE
Add a useful readme

### PR DESCRIPTION
**Pull request by kaldras**
Update the initial README skeleton into an actually useful README.

### CODE REVIEW

**Comment by hairmare** [comment]
Fixes #1

File: README.md [hunk 1]
```diff
@@ -1,50 +1,157 @@
...
+// app/adapters/application.js
+import DS from "ember-data";
+import OIDCAdapterMixin from "ember-simple-auth-oidc/mixins/oidc-adapter-mixin";
+
+export default DS.JSONAPIAdapter.extend(OIDCAdapterMixin, {});
+```
+
+## Configuration
+
+The addon can be configured in the ember config with the key `ember-simple-auth-oidc`.
```
Comment (czosel) [inline README.md]:
```suggestion
The addon can be configured in your `environment.js` file with the key `ember-simple-auth-oidc`.
```

File: README.md [hunk 2]
```diff
@@ -1,50 +1,157 @@
...
+    "ember-simple-auth-oidc": {
+      host: "http://authorization.server/openid",
+      clientId: "test",
+      authEndpoint: "/authorize",
+      tokenEndpoint: "/token",
+      userinfoEndpoint: "/userinfo",
+    }
+  }
+  return ENV;
+}
```
Comment (czosel) [inline README.md]: I'd focus on only the relevant section, maybe like this:
```js
module.exports = function(environment) {
  let ENV = {
    // ...
    "ember-simple-auth-oidc": {
      host: "http://authorization.server/openid",
      clientId: "test",
      authEndpoint: "/authorize",
      tokenEndpoint: "/token",
      userinfoEndpoint: "/userinfo",
    }
    // ...
  }
}
```

File: README.md [hunk 3]
```diff
@@ -1,50 +1,157 @@
...
+```
+
+A complete list of all possible config options can be found below:
```
Comment (czosel) [inline README.md]: I'd write it a bit more casually, maybe
```suggestion
Here's a complete list of all possible config options:
```

File: README.md (hunk 1 above)
Comment (kaldras) [reply README.md]: How about `in the projects environment.js file`? Because I never used you/your until now.

Comment (czosel) [reply README.md]: Sounds good! Nitpick: `in the project's`

Comment (kaldras) [reply README.md]: ok, done

File: README.md [hunk 4]
```diff
@@ -1,50 +1,157 @@
...
+import OIDCAdapterMixin from "ember-simple-auth-oidc/mixins/oidc-adapter-mixin";
+
+export default DS.JSONAPIAdapter.extend(OIDCAdapterMixin, {});
+```
+
+## Configuration
+
+The addon can be configured in the ember config with the key `ember-simple-auth-oidc`.
+
+At least the following config options need to be added to the ember config:
```
[the remark this answers is not in the document]
Comment (kaldras) [reply README.md]: Sounds better, changed this one as suggested

File: README.md (hunk 2 above)
Comment (kaldras) [reply README.md]: Agree, also changed this one as suggested.

File: README.md (hunk 3 above)
Comment (kaldras) [reply README.md]: Liked this one too, done

**Pull request merged**